README.md

```diff
@@ -1 +1,57 @@
-This is a starter template for [Learn Next.js](https://nextjs.org/learn).
\ No newline at end of file
+# 📜 The Introduction
+
+**Open Source Community Kampala** is an organisation for open-source lovers, enthusiasts, advocates, and experts within and across Uganda but centered in Kampala with the sole aim of increasing the rate of credible contributions by Ugandan software developers, designers, writers, and everyone involved in the sphere of technology to open source projects both locally and globally, changing the perception the world has of our developers; we are not just users of software tools, we are also **Software creators**.
+
+![](.gitbook/assets/osca-logo_colored.png)
+
+You are welcome to our documentation guide; if you are looking to join **OSCA-Kampala Chapter** and want to learn more about **Free and Open Source Software** and also want to know how you can make a contribution to any of our projects. Read this documentaion as a guide; Although it is a work in progress, please feel free to contribute and give us feedback to improve it.
+
+To suggest changes, click the GitHub icon on any docs page and make a pull request. If you're not sure how to do that, read our [documentation contribution guidelines](https://github.com/OSCA-Kampala-Chapter/First-contribution-practice/blob/main/CONTRIBUTING.md).
+
+If you need more help, please contact us at [community@oscakampala.org](mailto:community@oscakampala.org) or join the `#documentation` channel on our [Discord](https://discord.gg/pRJgjH9SwR) server.
```

Comment:
Our email is oscakampala@gmail.com, we currently do not have a domain of oscakampala.org

Reply:
I think we need to buy that domain; it looks good on us; 
so what should I add to the domain name part in the meantime?

Let me make changes to the email address

Reply:
Just leave the domain name part blank I guess. for now, we can edit that later